arvo/clay/data-types.md

```diff
@@ -216,91 +281,68 @@ that was at the head on date `p`, `%tas` refers to the commit labeled
+      hit=(map aeon tako)                               ::  versions by id
+      lab=(map @tas aeon)                               ::  labels
+      mim=(map path mime)                               ::  mime cache
+      fod=ford-cache                                    ::  ford cache
+      fer=(unit reef-cache)                             ::  reef cache
+  ==                                                    ::
 ```
 
-This is the data that is actually stored in a desk.
-
-`ang` is unused and should be removed.
+A `dome` is the state of a `desk` and associated data.
 
 `ank` is the current state of the desk. Thus, it is the state of the
-filesystem at revison `let`. The head of a desk is always a numbered
+filesystem at revison `let`. The head of a `desk` is always a numbered
 commit.
 
 `let` is the number of the most recently numbered commit. This is also
 the total number of numbered commits.
 
-`hit` is a map of numerical ids to hashes of commits. These hashes are
-mapped into their associated commits in `hut:rang:clay`. In general, the keys
-of this map are exactly the numbers from 1 to `let`, with no gaps. Of
-course, when there are no numbered commits, `let` is 0, so `hit` is
-null. Additionally, each of the commits is an ancestor of every commit
-numbered greater than this one. Thus, each is a descendant of every
-commit numbered less than this one. Since it is true that the date in
-each commit (`t:yaki`) is no earlier than that of each of its parents,
-the numbered commits are totally ordered in the same way by both
-pedigree and date. Of course, not every commit is numbered. If that
-sounds too complicated to you, don't worry about it. It basically
-behaves exactly as you would expect.
+`hit` is a map of numerical ids to commit hashes. These hashes are mapped into
+their associated commits in `hut.rang` in the `raft` of Clay. In general, the
+keys of this map are exactly the numbers from 1 to `let`, with no gaps. Of
+course, when there are no numbered commits, `let` is 0, so `hit` is null.
+Additionally, each of the commits is an ancestor of every commit numbered
+greater than this one. Thus, each is a descendant of every commit numbered less
+than this one. Since it is true that the date in each commit (`t:yaki`) is no
+earlier than that of each of its parents, the numbered commits are totally
+ordered in the same way by both pedigree and date. Of course, not every commit
+is numbered. If that sounds too complicated to you, don't worry about it. It
+basically behaves exactly as you would expect.
 
 `lab` is a map of textual labels to numbered commits. Note that labels
 can only be applied to numbered commits. Labels must be unique across a
 desk.
 
-#### `++ankh`, filesystem node
+`mim` is a cache of the content in the directories that are mounted to Unix.
+Often, we convert to/from mime without anything really having changed; this lets
+us short-circuit that in some cases. Whenever you `%give` an `%ergo` `gift`, you
```

Comment:
I don't really follow what you mean about %ergo.

Reply:
I mostly just copied from this comment:

```
::  `mim` is a cache of the content in the directories that are mounted
::  to unix.  Often, we convert to/from mime without anything really
::  having changed; this lets us short-circuit that in some cases.
::  Whenever you give an `%ergo`, you must update this.
```

Looking around a bit more, I get the impression that `%ergo` moves automatically update `mim`? So this is more of a reminder to the coder to make sure `mim` gets updated when messing with `%ergo`'s logic than a user. But this is just based off of 5 minutes of perusing - I could be totally off.

Reply:
I'd just remove everything about `mim` except the sentence saying it's a cache of the Unix mount data.  The rest of that comment is pretty deep into the internals and I think will mostly seem like noise from the perspective of someone first learning about Clay.

Reply:
https://github.com/urbit/docs/pull/1055/commits/3b8c67edb3c436a3be5a3405ed14283feba27e80